Werewolf for Telegram/Werewolf Control/Commands/AdminCommands.cs

```diff
@@ -305,7 +305,7 @@ public static void SetLink(Update update, string[] args)
             }
 
             var link = args[1].Trim();
-            if (!Regex.IsMatch(link, @"^(https?:\/\/)?t(elegram)?\.me\/joinchat\/([a-zA-Z0-9_\-]+)$"))
+            if (!Regex.IsMatch(link, @"^(https?:\/\/)?t(elegram)?\.me\/joinchat\/([a-zA-Z0-9_\-]+)$") || !Regex.IsMatch(link, @"^(https?:\/\/)?t(elegram)?\.me\/+([a-zA-Z0-9_\-]+)$"))
```

Comment:
This...

1. ...doesn't work, as the second regex is wrong. The `+` sign in ...`\.me\/+`... needs to be escaped, as it will be treated as a regex quantifier currently.
2. ...isn't elegant. *All* cases so far have been handled by a single regex, so you should put the alternative of `+` and `joinchat/` in the existing regex instead of creating a second one to check against.

Reply:
Appreciate the elaborate comments.
Have tested and updated the regex.